napari/_qt/widgets/qt_extension2reader.py

```diff
@@ -50,16 +52,10 @@ def __init__(self, parent=None, npe2_readers=None, npe1_readers=None):
 
         instructions = QLabel(
             trans._(
-                'Enter a filename pattern to associate with a reader e.g. "*.tif" for all TIFF files.'
-            )
-            + trans._(
-                'Available readers will be filtered to those compatible with your pattern. Hover over a reader to see what patterns it accepts.'
-            )
-            + trans._(
-                '\n\nPreference saving for folder readers is not supported, so these readers are not shown.'
-            )
-            + trans._(
-                '\n\nFor documentation on valid filename patterns, see https://docs.python.org/3/library/fnmatch.html'
+                'Enter a filename pattern to associate with a reader e.g. "*.tif" for all TIFF files. '
+                + 'Available readers will be filtered to those compatible with your pattern. Hover over a reader to see what patterns it accepts.'
+                + '\n\nYou can save a preference for a specific folder by listing the folder name with a "/" at the end (for example, "/test_images/").'
+                + '\n\nFor documentation on valid filename patterns, see https://docs.python.org/3/library/fnmatch.html'
```

Comment:
Trans requires single line strings. I know it will be big.

Reply:
ok.  will make the change.